### PR TITLE
[TIR] Introduce Pass UnifyKernelLaunch and speedup split op

### PR DESCRIPTION
**Pull request by Hzfengsy**
This PR introduces a new pass UnifyKernelLaunch to fuse the nearby kernels with same thread config, which will reduce the kernel launch overhead in `split` operator.

Test on the following settings:

```python
A = te.placeholder((75, 182), "int64")
B = topi.split(A, 182, axis=1)

func = te.create_prim_func([A, *B])
```

The pass will reduce the latency from 347.03 us to 5.22 us on my machine.

cc @MasterJH5574 @vinx13 @junrushao

### CODE REVIEW

**Comment by Hzfengsy** [comment]
@tvm-bot rerun

File: src/meta_schedule/schedule/cuda/split.cc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+namespace meta_schedule {
+
+using namespace tvm::tir;
+
+TVM_REGISTER_GLOBAL("meta_schedule.cuda.split")
+    .set_body_typed([](Schedule sch, BlockRV data_pack) -> Array<Schedule> {
+      // Do nothing, which ensure the same thread config for each block.
+      // We can fuse these kernels into only one.
+      return {sch};
+    });
```
Comment (junrushao) [inline src/meta_schedule/schedule/cuda/split.cc:31]: if we want to skip a block, use {"schedule_rule": "None"} instead

Comment (Hzfengsy) [reply src/meta_schedule/schedule/cuda/split.cc:31]: We only want to skip the block for GPU backend, but still will apply schedule on other targets.

Comment (junrushao) [reply src/meta_schedule/schedule/cuda/split.cc:31]: perhaps we should support per-device None in meta schedule :-)

**Review by vinx13 — APPROVED** [review]

**Pull request closed without merging**